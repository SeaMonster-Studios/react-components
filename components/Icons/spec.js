// @flow
import React from 'react'
import { render, cleanup } from 'react-testing-library'
//
import { ArrowDownIcon } from './'

afterEach(cleanup)

describe('Icons Tests', () => {
  it('Renders ArrowDownIcon', () => {
    const { container } = render(<ArrowDownIcon />)

    expect(container.firstChild).toMatchSnapshot()
  })
})
