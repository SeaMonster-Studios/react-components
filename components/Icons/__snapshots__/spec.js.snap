// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icons Tests Renders ArrowDownIcon 1`] = `
<svg
  fill="none"
  height="10"
  viewBox="0 0 18 10"
  width="18"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M8.02383 9.76252L0.236637 1.85449C-0.0788788 1.53898 -0.0788788 1.02878 0.236637 0.713267L0.713266 0.236637C1.02878 -0.0788788 1.53898 -0.0788788 1.85449 0.236637L8.59444 7.10413L15.3344 0.24335C15.6499 -0.0721653 16.1601 -0.0721653 16.4756 0.24335L16.9522 0.719979C17.2678 1.03549 17.2678 1.54569 16.9522 1.86121L9.16505 9.76923C8.84954 10.078 8.33934 10.078 8.02383 9.76252Z"
    fill="#0072CE"
  />
</svg>
`;
